intel/auto-pts
# wid: Add support for L2CAP/LE/CFC/BV-25-C

The PTS software (8.2.0) issues are fixed and expected to released
in next release.The new steps are added to the test to fullfil the
test specification properly.

Test steps:
1-IUT creates connection to PTS (PTS should be advertising)
2-PTS sends security request
3-IUT sends pairing request
4-Pairing complete (with bonding option, so LTK will be stored for coming
  connections)
- new step: Disconnect (WID=272)
- new step: Reconnect: IUT creates connection to PTS (same as step1)
- new WID=138 to make sure L2CAP channel encryption requrements before
the following request.

5-PTS sends L2CAP_LE_CREDIT_BASED_CONNECTION_REQ
6-IUT is expected to respond with result code as 0x08=Insufficient Encryption

See the PTS software case (CASE0072529) for more details. The fix patch
is shared as L2CAP.ets
